### PR TITLE
docs: docs for prism anoncreds method support

### PR DESCRIPTION
**Pull request by shotexa**
This PR should be merged together with https://github.com/hyperledger/identus-cloud-agent/pull/1386

### CODE REVIEW

File: documentation/docs/concepts/glossary.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ The process for retrieving a [DID document](#did-document).
 ### DID subject
 The entity is identified by a [DID](#decentralized-identifer-(did)) and described by a [DID documents](#did-document). Anything can be a DID subject: person, group, organization, physical thing, digital thing, etc.
 
+### DID Url
+A DID itself is a type of a URL, while `did` is a registered schema type, like `http` and `https`. In identus system, we refer DID urls to a DID that includes path and query paramteres and can resolve a resource via one of the service endpoints in the DID document. for example:
```
Comment (petevielhaber) [inline documentation/docs/concepts/glossary.md:71]:
```suggestion
A DID itself is a type of a URL, while `did` is a registered schema type, like `http` and `https`. With Identus, we refer DID URLs to a DID that includes path and query parameters and can resolve a resource via one of the service endpoints in the DID document. For example: 
```

**Review by petevielhaber — APPROVED** [review]
- Made small adjustment

**Review by yshyn-iohk — APPROVED** [review]

**Comment by yshyn-iohk** [comment]
@shotexa, are you sure that you need to commit a `package.json` file?

**Comment by shotexa** [comment]
> @shotexa, are you sure you must commit a `package.json` file?

I could not run "npm start" without that modification which is why I've committed it for others.

**Pull request merged**